lib/api/admin_api.ts

```diff
@@ -497,6 +497,25 @@ export const specs = {
         "authorized_recipients"
       ]
     },
+    ServicePublic: {
+      title: "Service (public)",
+      description: "A Service tied to an user's subscription.",
```

Comment:
`tied` has a literal meaning (having one's hands tied), perhaps `associated` would be better in this context